tests/.python-version

```diff
@@ -0,0 +1,8 @@
+3.11.3
```

Comment:
If only there was a way of omitting the patch version so that we can stay up-to-date. I think `:latest` only works with the install command